dbaccess/block.go

```diff
@@ -115,21 +108,16 @@ func SelectedTipBlueScore(ctx Context) (uint64, error) {
 		return 0, err
 	}
 
-	var blueScore []uint64
-	dbResult := db.Model(&dbmodels.Block{}).
-		Where(&dbmodels.Block{IsChainBlock: true}).
-		Select("MAX(`blue_score`) as `blue_score`").
-		Pluck("blue_score", &blueScore)
-
-	dbErrors := dbResult.GetErrors()
-	if httpserverutils.HasDBError(dbErrors) {
-		return 0, httpserverutils.NewErrorFromDBErrors("some errors were encountered when loading selected tip blue score from the database:", dbErrors)
+	var blueScore uint64
+	err = db.Model(&dbmodels.Block{}).
+		Where("is_chain_Block = ?", true).
+		ColumnExpr("MAX(blue_score) as blue_score").
+		Select(&blueScore)
```

Comment:
Shouldn't you use .First or something like this?

Reply:
No, MAX queries returns a number (maximum value of a column)